docker/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN dnf -y install \
   --setopt=tsflags=nodocs \
   # Install brewkoji for the koji.conf.d config file
   brewkoji \
+  git \
```

Comment:
Could you use `git-core` instead? It should make the image smaller.

Reply:
Nice idea